ethportal-peertest/src/scenarios/basic.rs

```diff
@@ -27,10 +27,7 @@ pub async fn test_discv5_node_info(peertest: &Peertest) {
 pub async fn test_discv5_routing_table_info(target: &Client) {
     info!("Testing discv5_routingTableInfo");
     let result = Discv5ApiClient::routing_table_info(target).await.unwrap();
-    let local_key = result.get("localNodeId").unwrap();
-    assert!(local_key.is_string());
-    assert!(local_key.as_str().unwrap().starts_with("0x"));
-    assert!(result.get("buckets").unwrap().is_array());
```

Comment:
The assertion previously checked that the `buckets` field in the returned `Value` was an array. The rationale for its removal is that it's no longer needed now that `RoutingTableInfo` is a concrete type since successful deserialization proves that it fits the defined structure.